docs/pages/setup/helm-reference/teleport-cluster.mdx

```diff
@@ -63,13 +63,26 @@ This reference details available values for the `teleport-cluster` chart.
 
 `kubeClusterName` sets the name used for the Kubernetes cluster. This name will be shown to Teleport users connecting to the cluster.
 
-## `authenticationType`
+## `authentication`
+
+### `authentication.type`
 
 | Type | Default value | Required? | `teleport.yaml` equivalent | Can be used in `custom` mode? |
 | - | - | - | - | - |
 | `string` | `local` | Yes | `auth_service.authentication.type` | ❌ |
 
-`authenticationType` controls the authentication scheme used by Teleport. Possible values are `local` and `github` for OSS, plus `oidc`, `saml`, and `false` for Enterprise.
+`authentication.type` controls the authentication scheme used by Teleport. Possible values are `local` and `github` for OSS, plus `oidc`, `saml` for Enterprise.
+
+## `authentication.localAuth`
+
+| Type | Default value | Required? | `teleport.yaml` equivalent | Can be used in `custom` mode? |
+| - | - | - | - | - |
+| `bool` | `true` | No | `auth_service.authentication.local_auth` | ❌ |
+
+`authentication.localAuth` controls whether local authentication is enabled.
+When disabled, users can only login through non-local auth like `saml`, `oidc` or `github`.
```

Comment:
```suggestion
When disabled, users can only log in through authentication connectors like `saml`, `oidc` or `github`.
```
"login" is a noun. Also making the language consistent with the rest of our docs.